src/admin-views/widgets/list.php

```diff
@@ -23,6 +23,21 @@
 }
 
 foreach ( $admin_fields as $field_id => $field ) {
+	// Handle a section start. May contain a section title.
+	// Using stripos() to allow for multiple occurrences ("section_start_1", "section_start_2" etc).
+	if ( false !== stripos( $field_id, 'section_start' ) ) : ?>
+		<div class="tribe-events-widget-admin-form__input-section">
+			<?php if ( ! empty( $field['title'] ) ) : ?>
+				<h4><?php echo esc_html( $field['title'] ); ?></h4>
+			<?php endif;
+		continue;
+	endif;
+
+	// Handle a section end.
+	if ( false !== stripos( $field_id, 'section_end' ) ) : ?>
+		</div>
+		<?php continue;
```

Comment:
:no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: dff4ad5b8175557e5e28e274020a7a4d</sub>